backend/src/test/unimplemented_resources/ID024_accessCreatedIdeas.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Access Created Ideas
+  As a user, I want to see all ideas that I have created so that I can recall what ideas I have put on the app
```

Comment:
As a user who has created an idea, I want to be able to access all the ideas I have created so that I can keep track of them and their associated collaboration requests

From backlog to match :)